app/components/select-input/template.hbs

```diff
@@ -1,6 +1,6 @@
 <select class="form-control auto-width" onchange={{action (mut value) value="target.value"}}>
-  <option value=null>Please select</option>
+  <option value=null>{{t 'global.selectUnselected'}}</option>
   {{#each options as |option|}}
-    <option value={{option}} selected={{eq value option}}>{{option}}</option>
+    <option value={{option}} selected={{eq value option}}>{{t option}}</option>
```

Comment:
It works! Although, currently users cannot go back in the survey so it's not necessary.